assets/js/map.js

```diff
@@ -0,0 +1,86 @@
+
+var mapOptions = {
+    center: [44.8, -2.46],
+    zoom: 3
+}
+
+// instantiating the map with user defined mapOptions passed as parameter
+var map = new L.map("map", mapOptions);
+
+// instantiating the TileLayer Class which displays the desired map UI (passed as a String parameter)
+var layer = new L.tileLayer('https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png');
+
+// add our custom layer to the map object created above
+map.addLayer(layer);
+
+const fellows = [{
```

Comment:
While this works, I think it would be best to store this information in the data files.

Reply:
Umm, then may be when #25 will be merged, then all the rqd data will automatically be there that ill need. So i wont need to add anything else. So, ill fetch the data from there only once I pull all the changes on my local after issue 25 is merged.

Reply:
Alright, #25 was just merged so you should be good to finish this up!

Good luck!